src/Vinelab/Cdn/CdnFacade.php

```diff
@@ -103,6 +103,27 @@ public function elixir($path)
         throw new \InvalidArgumentException("File {$path} not defined in asset manifest.");
     }
 
+    /**
+     * This function allow Laravel Mix to call the right rev-manifest for assets.
+     *
+     * @param $path
+     *
+     * @return mixed
+     *
+     * @throws Eceptions\EmptyPathException, \InvalidArgumentException
+     */
+    public function mix($path)
+    {
+        static manifest = null;
+        if (is_null($manifest)) {
```

Comment:
Why check if `$manifest` is null ? `$manifest` is set to null just before

Reply:
It was a c/c from the elixir() method :)